charts/umbrella/templates/configmap-backend-seeding-testdata.yaml

```
@@ -0,0 +1,259 @@
+{{- /*
+* Copyright (c) 2024 Contributors to the Eclipse Foundation
+*
+* See the NOTICE file(s) distributed with this work for additional
+* information regarding copyright ownership.
+*
+* This program and the accompanying materials are made available under the
+* terms of the Apache License, Version 2.0 which is available at
+* https://www.apache.org/licenses/LICENSE-2.0.
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+* WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+* License for the specific language governing permissions and limitations
+* under the License.
+*
+* SPDX-License-Identifier: Apache-2.0
+*/}}
+
+{{- if .Values.portal.backend.portalmigrations.seeding.useDefault -}}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name:  "portal-default-testdata"
+  namespace: {{ .Release.Namespace }}
+data:
+  addresses.defaulttest.json: |-
```

could you please always change to test, also for all the other tables in the configMap
```suggestion
  addresses.test.json: |-
```